packages/block-editor/src/components/block-content-overlay/index.js

```diff
@@ -2,114 +2,27 @@
  * WordPress dependencies
  */
 import { useSelect } from '@wordpress/data';
-import { useState, useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { store as blockEditorStore } from '../../store';
 
-/**
- * External dependencies
- */
-import classnames from 'classnames';
-
-export default function BlockContentOverlay( {
```

Comment:
Should we deprecate? Directory search return single plugin - https://wpdirectory.net/search/01G29EAZQER3R2XCAGSMB7MRD5

Reply:
this was never stable in the first place. So I think it's fine to just remove.

Reply:
(The result in wp-directory is a plugin that show up in all Gutenberg API searches :P I think it's bundling our packages)